src/media/js/apps_buttons.js

```diff
@@ -47,29 +48,23 @@ define('apps_buttons',
 
         // If it's a paid app, ask the user to sign in first.
         if (product.receipt_required && !user.logged_in()) {
-            var payWindow;
+            console.log('Install suspended; user needs to log in');
 
-            if (!capabilities.navPay) {
-                console.log('Creating payment window ahead of time via user click');
-                payWindow = payments.utils.openWindow({url: ''});
-            }
+            // Create a blank window here so we can pass it to the login func;
+            loginPopup = (!capabilities.navPay) ? utils.openWindow() : undefined;
```

Comment:
when the function is re-entered from ``login.done(...)`` then won't this open a new window? Shouldn't it be more like:
````
if (!loginPopup) {
    loginPopup = (!capabilities.navPay) ? ...
}
````
but maybe I'm missing something. It seems weird to call install() multiple times.